src/main/js/utils/grade.js

```diff
@@ -12,36 +12,36 @@ export const getInstructorLastName = instructor => instructor.split(' ').pop();
 
 const getMapping = (gradeList) => {
   const map = {};
-  /** map array of grades to its respective course-instructor */
   gradeList.forEach((course) => {
     const name = `${course.subject}${course.courseNumber}`;
 
     if (!map[name]) {
       map[name] = {};
-      map[name][course.instructor] = [course.gpa];
-    } else if (!map[name][course.instructor]) {
-      map[name][course.instructor] = [course.gpa];
+    }
+
+    if (!map[name][course.instructor]) {
+      map[name][course.instructor] = { qualityCredits: course.gpa * course.credits, credits: course.credits }; // eslint-disable-line max-len
```

Comment:
Only disable when it's something you have to do / can't get around

Reply:
Fixed.